<ui version="4.0">
 <class>add_new_gestao</class>
 <widget name="add_new_gestao" class="QDialog">
 <property name="geometry">
  <rect>
   <x>0</x>
   <y>0</y>
   <width>400</width>
   <height>207</height>
  </rect>
 </property>
 <property name="windowTitle">
  <string>Dialog</string>
 </property>
 <widget class="QPushButton" name="btn_salvar">
  <property name="geometry">
   <rect>
    <x>160</x>
    <y>170</y>
    <width>80</width>
    <height>21</height>
   </rect>
  </property>
  <property name="text">
   <string>Gravar</string>
  </property>
 </widget>
 <widget class="QWidget" name="">
  <property name="geometry">
   <rect>
    <x>30</x>
    <y>40</y>
    <width>351</width>
    <height>101</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nome:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_nome"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Endere√ßo:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_end"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Telefone:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_tel"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
</widget>
<resources/>
<connections/>
</ui>
